ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/activity-blocks/modalActivityBlock/modalActivityBlock.component.html

```diff
@@ -11,9 +11,18 @@
 
             <div *ngIf="instance.statusCode !== 'CREATED'"
                  class="modal-activity-block__success-message">
-                <mat-icon inline>check_circle</mat-icon>
-                <span [innerHTML]="'SDK.ModalActivityBlock.NumQuestionsAnswered'
-          | translate: { number: instance.numQuestionsAnswered }"></span>
+                <mat-icon *ngIf="instance.numQuestionsAnswered === instance.numQuestions" inline>check_circle</mat-icon>
```

Comment:
**_As an option_**, keep this check as a boolean getter in the component, like e.g. `isAllQuestionsCompleted`

Reply:
good idea, done!